pkg/flink/internal/controller/interactive_output_controller.go

```diff
@@ -200,14 +200,15 @@ func (t *InteractiveOutputController) getTableTitle() string {
 
 	if t.debug {
 		return fmt.Sprintf(
-			" %s | last page size: %d | current cache size: %d/%d | table size: %d ",
+			" %s (%s) | last page size: %d | current cache size: %d/%d | table size: %d ",
 			mode,
+			t.resultFetcher.GetStatement().StatementName,
 			t.resultFetcher.GetStatement().GetPageSize(),
 			t.resultFetcher.GetMaterializedStatementResults().GetChangelogSize(),
 			t.resultFetcher.GetMaterializedStatementResults().GetMaxResults(),
 			t.resultFetcher.GetMaterializedStatementResults().GetTableSize(),
 		)
 	}
 
-	return fmt.Sprintf(" %s ", mode)
+	return fmt.Sprintf(" %s (%s) ", mode, t.resultFetcher.GetStatement().StatementName)
```

Comment:
```suggestion
	return fmt.Sprintf(" %s (%s) ", mode, t.resultFetcher.Statement.GetStatementName())
```

Reply:
I can't access it like that, because `statement` is private and a shared resource. The getter function is used to synchronize access to it to avoid races:
https://github.com/confluentinc/cli/blob/6a3583314693136dc7a7e1324e2a75b3412e6a79/pkg/flink/internal/results/result_fetcher.go#L144-L149